app/javascript/src/NewService/components/FormElements/FormWrapper.jsx

```diff
@@ -0,0 +1,33 @@
+// @flow
+
+import React from 'react'
+import type {FormProps} from 'NewService/types'
```

Comment:
But it's also used [here](https://github.com/3scale/porta/pull/763/files?file-filters%5B%5D=.feature&file-filters%5B%5D=.js&file-filters%5B%5D=.json&file-filters%5B%5D=.jsx&file-filters%5B%5D=.slim#diff-f4cc5a54f4c8e4f49ebbbe480646b19aR9)

Reply:
True, but it's not necessary. In case it was, I'd export it from this file.

Reply:
I think is better to have it in its own file, because it doesn't belongs exclusively to `FormWrapper`, it's also used in `ServiceDiscoveryForm` and `ServiceManualForm`. For me it's more clear importing types for a types file than from a component